test/restaurant_inspections_api/handlers/counties_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns restaurant-inspections-api.handlers.counties-test
+  (:require [restaurant-inspections-api.handlers.counties :refer :all]
+            [clojure.test :refer :all]))
+
+(def ctx-mock {:request {:params {}}})
+(def response-mock {:meta {}
+                    :data [{:district "D1"
+                            :countynumber 23
+                            :countyname "Dade"
+                            :inspections 20630}]})
+;(deftest handle-ok-test
```

Comment:
and also for some reason, the test-refresh module I use locally did not throw errors while Travis CI did. I need to configure the test runner we have in Travis to watch the files and run the tests for the changed files